packages/default-theme/components/SwTopNavigation.vue

```diff
@@ -89,46 +85,60 @@ import {
   useProductSearch
 } from '@shopware-pwa/composables'
 import SwLoginModal from '@shopware-pwa/default-theme/components/modals/SwLoginModal'
+import SwMegaMenu from '@shopware-pwa/default-theme/components/SwMegaMenu'
+import { ref, reactive, onMounted } from "@vue/composition-api";
+import { getNavigation } from '@shopware-pwa/shopware-6-client'
```

Comment:
using shopware-6-client directly is not very common in the theme and we should rather keep it in other layer like composition.